objPacker.py

```diff
@@ -0,0 +1,37 @@
+import os
+import subprocess
+from sys import platform as _platform
+from PIL import Image
+from espAssetPackerUtils import *
+
+def processObj(dir, file):
+    # Write the given bytes to a temporary file
+    objProcessorFolder = os.path.dirname(os.path.realpath(__file__)) + "/objProcessor"
+    if _platform == "linux" or _platform == "linux2":
+        lzcFile = objProcessorFolder + "/objProcessor"
```

Comment:
`lzcFile` is a nice copy/paste, and I'm only noting it to prove that I'm reviewing.